CONTRIBUTING.rst

```diff
@@ -348,17 +348,15 @@ These standards will be enforced using ``pre-commit`` using
 If for whatever reason you feel that your function should have an exception to
 any of the rules, add an exception to the function either in the
 ``[tool.numpydoc_validation]`` section in ``pyproject.toml`` or add an inline
```

Comment:
The changes in this module also don't look auto-generated (concerning https://github.com/pyvista/pyvista/pull/6842#discussion_r1854344488).

Reply:
Correct. These changes were added after that comment was made.

Reply:
OK. I'm resolving that thread in that case.